File: 2021/05/24/main.go
package main

func toLowerCase(s string) string {
	r := ""
	for i := 0; i < len(s); i++ {
		c := s[i]
		if 'A' <= c && c <= 'Z' {
			c = c + 'a' - 'A'
		}
		r += string(c)
	}
	return r
}

File: 2021/05/24/main.py
class Solution:
    def toLowerCase(self, s: str) -> str:
        r = ''
        for i in range(len(s)):
            c = s[i]
            if 'A' <= c and c <= 'Z':
                c = chr(ord(s[i]) + ord('a') - ord('A'))
            r += c
        return r